aioshelly/rpc_device.py

```diff
@@ -2,18 +2,21 @@
 from __future__ import annotations
 
 import asyncio
-from typing import Any, Dict
+from typing import Any, Callable
 
 import aiohttp
+from aiohttp.client import ClientSession
 
 from .common import ConnectionOptions, IpOrOptionsType, get_info, process_ip_or_options
-from .exceptions import AuthRequired, NotInitialized, WrongShellyGen
+from .exceptions import AuthRequired, NotInitialized, ShellyError, WrongShellyGen
```

Comment:
It is better to use the `AuthRequired` as we use in other places and not generic error
```suggestion
from .exceptions import AuthRequired, NotInitialized, WrongShellyGen
```

Reply:
IMHO, If Shelly is missing is not a auth error, but a generic error